src/SpriteAnimation/SpriteAnimation.js

```diff
@@ -493,8 +502,14 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	_generateKeyframe: function (percent, x, y) {
-		return (Math.ceil(percent*1000000) / 10000) +'%	{ -webkit-transform: translate3d('+ x +'px, '+ y +'px, 0);	transform: translate3d('+ x +'px, '+ y +'px, 0); }\n';
+	_generateKeyframe: function (percent, x, y, prefix) {
+		var outStr = (Math.ceil(percent*1000000) / 10000) +'%	{ ';
+		if (prefix) {
+			// add the prefix and the standard
+			outStr+= prefix + 'transform: translate3d('+ x +'px, '+ y +'px, 0); ';
```

Comment:
What do you think about throwing `'transform: translate3d('+ x +'px, '+ y +'px, 0); ` into a var so we don't have to repeat the concatenation?